semantic_router/utils/splitters.py

```diff
@@ -0,0 +1,97 @@
+from typing import Any, List, Literal, Optional
```

Comment:
this file `/utils/splitters.py` can be removed as it now exists in the `/splitters` module